docs/source/users/index.md

```diff
@@ -357,6 +357,10 @@ GPT4All support is still an early-stage feature, so some bugs may be encountered
 during usage. Our team is still actively improving support for locally-hosted
 models.
 
+### Ollama usage
+
+To get started, follow the instructions on the [Ollama website](https://ollama.com/) to set up `ollama` and download the models locally. To select a model, enter the model name in the settings panel, for example `deepseek-coder-v2`.
+
```

Comment:
Suggest adding a few more notes to the docs as follows:

Once `Ollama` is installed, download the model using the following command in a terminal:
```
ollama pull <model_name>
``` 
For example: `ollama pull llama3`. Do this for both LLM models and embedding models, as many as needed. 

Once the models are downloaded, start up the `Ollama` server:
```
ollama serve
```

The models will then be available in `jupyter-ai`.

Reply:
Good suggestion, but wouldn't block on this to merge. Also, this info can change based on `ollama` API, so we will be carrying instructions that are available in their docs and will have to update in case this changes.

Reply:
@3coins Agree. I tested it again and it's all working so merging is predicated.